boot/zephyr/Kconfig

```diff
@@ -58,6 +58,14 @@ config NRF_CC310_BL
 
 menu "MCUBoot settings"
 
+config SINGLE_IMAGE_DFU
```

Comment:
This is equivalent to `MCUBOOT_IMAGE_NUMBER == 0.5 true case`. Shouldn't be ensure that MCUBOOT_IMAGE_NUMBER is equal to 1?

Reply:
It could be checked.

Reply:
Do you want the check to be done here boot/zephyr/include/target.h, same way the other checks are done?

Reply:
OK, done it via limiting range on `UPDATEABLE_IMAGE_NUMBER`, in Kconfig, when `SINGLE_IMAGE_DFU` is selected.